### PR TITLE
Adding Github Teams to Working Group doc

### LINKED ISSUE #450: Update the WG doc to include the Github team alias
Update the [WG doc](https://github.com/knative/community/blob/master/working-groups/WORKING-GROUPS.md) to include the Github team alias `@<wg>` 

This would make it easier to discover how to @ a team in an issue or PR, for example in docs we need to assign PRs that need the WG to take care of the PR to resolution and for new members to the community, they might not know what's the team alias.

Take the current info from [peribolos](https://github.com/knative/community/blob/master/peribolos/knative.yaml)

cc @evankanderson

### PR DESCRIPTION
**Pull request by csantanapr**
Closes #450

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -162,6 +167,7 @@ Event delivery data plane.
 | Document Folder            | [Folder](https://drive.google.com/drive/folders/1YcyeckwBNPApxh8vHAlscYe_w7sIA5TA)                                                                          |
 | Repos                      | [`knative/eventing`](https://github.com/knative/eventing), [`knative/eventing-contrib`](https://github.com/knative/eventing-contrib), (shares `eventing-*`) |
 | Slack Channel              | [#eventing-delivery](https://slack.knative.dev/messages/eventing-delivery)                                                                                  |
+| Github Team WG leads       | [@knative/eventing-wg-leads](https://github.com/orgs/knative/teams/eventing-wg-leads/members)                                                               |
```
Comment (mattmoor) [inline working-groups/WORKING-GROUPS.md:170]: https://github.com/orgs/knative/teams/channel-wg-leads

Comment (csantanapr) [reply working-groups/WORKING-GROUPS.md:170]: Thanks I couldn't delivery WG lead, so it's channel I guess

Comment (csantanapr) [reply working-groups/WORKING-GROUPS.md]: pushed the fix

**Comment by csantanapr** [comment]
/assign @mattmoor 
😉

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**